js/event/singin.js

```diff
@@ -30,16 +29,20 @@ const loginBtn = document.querySelector('.boxmain');
   });
 
     // 마우스 클릭으로 로그인 실행
-  loginBtn.addEventListener('click', loginAttempt);
+  loginBtn.addEventListener('click', function(){
+    loginAttempt(id.emailInput.value, pw.passwordInput.value)
+    errorMsg.clearErrors()
+  });
```

Comment:
r: removeEventListener로 이벤트 핸들러 제거 시 addEventListener로 등록한 동일한 함수를 인자로 전달해야하기 때문에 함수를 먼저 선언하고 등록하는 것을 추천합니다.
```suggestion
  function handleClickLoginButton() {
    loginAttempt(id.emailInput.value, pw.passwordInput.value)
    errorMsg.clearErrors()
  }

  loginBtn.addEventListener('click', handleClickLoginButton);
```